packages/@aws-cdk/cicd/README.md

```diff
@@ -0,0 +1,112 @@
+## Continuous Integration / Continuous Delivery for CDK Applications
+This library includes a *CodePipeline* action for deploying AWS CDK Applications.
+
+### Getting Started
+In order to add the `PipelineDeployStackAction` to your *CodePipeline*, you need to have a *CodePipeline* artifact that
+contains the result of invoking `cdk synth -o <dir>` on your *CDK App*. You can for example achieve this using a
+*CodeBuild* project.
+
+The example below defines a *CDK App* that contains 3 stacks:
+* `CodePipelineStack` manages the *CodePipeline* resources, and self-updates before deploying any other stack
+* `ServiceStackA` and `ServiceStackB` are service infrastructure stacks, and need to be deployed in this order
+
+```
+  ┏━━━━━━━━━━━━━━━━┓  ┏━━━━━━━━━━━━━━━━┓  ┏━━━━━━━━━━━━━━━━┓  ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+  ┃     Source     ┃  ┃     Build      ┃  ┃  Self-Update   ┃  ┃            Deploy             ┃
+  ┃                ┃  ┃                ┃  ┃                ┃  ┃                               ┃
+  ┃ ┌────────────┐ ┃  ┃ ┌────────────┐ ┃  ┃ ┌────────────┐ ┃  ┃ ┌────────────┐ ┌────────────┐ ┃
+  ┃ │   GitHub   ┣━╋━━╋━▶ CodeBuild  ┣━╋━━╋━▶Deploy Stack┣━╋━━╋━▶Deploy Stack┣━▶Deploy Stack│ ┃
+  ┃ └────────────┘ ┃  ┃ └────────────┘ ┃  ┃ └────────────┘ ┃  ┃ └────────────┘ └────────────┘ ┃
```

Comment:
Can you add `CodePipelineStack`, `ServiceStackA` and `ServiceStackB` Stack names to the diagram? Perhaps under the `Deploy Stack'  label in the box. I think it will make it more clear.